base/src/base_layers.ts

```diff
@@ -1727,9 +1727,6 @@ export function labels_layers(source: string, c: Theme): any[] {
       "source-layer": "places",
       filter: ["==", "pmap:kind", "locality"],
       layout: {
-        "symbol-sort-key": ["to-number", ["concat", ["get", "pmap:min_zoom"],
```

Comment:
This becomes redundant because the logic is the same as the sorting of the tiles internally in java.

Reply:
Nice! I've applied sortKety to other label related tile layers, like POIs and Roads in subsequent comments. I'll 2x check the style layers are also updated...